File: examples/python/tasks/text_generation/text_generation.py
from bytez import Bytez

client = Bytez("YOUR BYTEZ KEY HERE")

model_params = {"max_new_tokens": 500, "min_new_tokens": 50, "temperature": 0.5}

model = client.model("Qwen/Qwen2-7B-Instruct")
model.load()
stream = model.run(
    "Once upon a time there was a beautiful home where",
    stream=True,
    model_params=model_params,
)

for chunk in stream:
    print(f"Output: {chunk}")

File: examples/julia/tasks/text_generation/text_generation.jl
using Bytez

client = Bytez.init("YOUR_BYTEZ_KEY_HERE")

model = client.model("Qwen/Qwen2-7B-Instruct")

model.load()

input_text = "Once upon a time there was a beautiful home where"

options = Dict(
	"params" => Dict(
		"max_new_tokens" => 500,
		"min_new_tokens" => 50,
		"temperature" => 0.5,
	),
	"stream" => true,
)

stream = model.run(input_text, options)

while isopen(stream)
	item = take!(stream)
	println(item)
end
